workflow-templates/techdocs.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish TechDocs Site
+
+on:
+  push:
+    branches: [main]
```

Comment:
Wouldn't it be better to reference $default-branch rather than restricting the branch to main?